src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -724,7 +724,7 @@ public function iSpecifyTheirShippingAddressAsFor($city, $street, $postcode, $co
     }
 
     /**
-     * @Then /^I should be notified that the ([^"]+) is required$/
+     * @Then /^I should be notified that the (?P<element>(last name|first name|city|street)) is required$/
```

Comment:
Can you check if [this](https://github.com/Sylius/Sylius/pull/6166/files/b98fb754a6476b9a35f1e59fc4dc07f7c7e54835#diff-3ae4eaa687e71825a6e859faa2a4e433R167) work fine?